### DISCUSSION TITLE
Contribution idea / personnal project for ETS2's steam success

### DISCUSSION THREAD

**Discussion by SylvianBurn** [Lihis/ets2-job-logger#65] [Q&A]
Hello,
First of all, as a ETS2 fan, i'd like to tell you that i really like your project :)

I would like to know if you think that i can use your api to monitor datas about jobs to help me/players to achieve some ETS2's steam success. I would like to know if you think it's better i fork this repo to do my stuff or that i try to contribute to it ?

Have a good day

**Comment by Lihis** [comment]
Application can be used of course, just implement the API on your end accroding the documentation and you're good to go.

I'm happy to take in contributions! So feel free to send PR's.
